lib/internal/bootstrap/loaders.js

```diff
@@ -211,8 +211,7 @@
 
     NativeModule.isInternal = function(id) {
       return id.startsWith('internal/') ||
-          (id === 'worker_threads' &&
-           !internalBinding('options').getOptions('--experimental-worker'));
+          (id === 'worker_threads' && !config.experimentalWorker);
```

Comment:
Can we avoid going back to `process.config`? :confused:

Reply:
There are also two other reference to `config` in here, I think a better option would probably be to just specialize something for NativeModule here - maybe just directly generate the whole map when the process starts and pass it to the loader bootstrapper here and let NativeModule short-circuit the `internal/options` implementation here. That also opens up the opportunity to emit all the warning for experimental modules in one place here instead of emitting them everywhere across the code base.